app/static/js/base.js

```diff
@@ -15,8 +16,11 @@ $(document).ready(function() {
     $("#userSelect").on('change', function() {
         $("#userSelectForm").submit();
     });
+    $("#flash_container").delay(5000).fadeOut();
```

Comment:
The trouble with this is that no more flashes will be able to show up until the page is reloaded. You need to fade just the alerts, not the container that holds them